test/e2e/util_gatewayapi_test.go

```diff
@@ -741,121 +698,88 @@ func assertCatalogSource(t *testing.T, namespace, csName string) error {
 
 	err := wait.PollUntilContextTimeout(context.Background(), 1*time.Second, 30*time.Second, false, func(context context.Context) (bool, error) {
 		if err := kclient.Get(context, nsName, catalogSource); err != nil {
-			t.Logf("failed to get catalogSource %s: %v, retrying...", csName, err)
+			t.Logf("Failed to get CatalogSource %s: %v.  Retrying...", csName, err)
 			return false, nil
 		}
 		if catalogSource.Status.GRPCConnectionState != nil && catalogSource.Status.GRPCConnectionState.LastObservedState == "READY" {
-			t.Logf("found catalogSource %s with last observed state %s", catalogSource.Name, catalogSource.Status.GRPCConnectionState.LastObservedState)
+			t.Logf("Found CatalogSource %s with last observed state: %s", catalogSource.Name, catalogSource.Status.GRPCConnectionState.LastObservedState)
 			return true, nil
 		}
-		t.Logf("found catalogSource %s but could not determine last observed state, retrying...", catalogSource.Name)
+		t.Logf("Found CatalogSource %s but could not determine last observed state.  Retrying...", catalogSource.Name)
 		return false, nil
 	})
 	return err
 }
 
-// assertSMCP checks if the ServiceMeshControlPlane exists in a ready state,
+// assertIstio checks if the Istio exists in a ready state,
 // and returns an error if not.
-func assertSMCP(t *testing.T) error {
+func assertIstio(t *testing.T) error {
 	t.Helper()
-	smcp := &maistrav2.ServiceMeshControlPlane{}
-	nsName := types.NamespacedName{Namespace: operatorcontroller.DefaultOperandNamespace, Name: openshiftSMCPName}
+	istio := &sailv1.Istio{}
+	nsName := types.NamespacedName{Namespace: operatorcontroller.DefaultOperandNamespace, Name: openshiftIstioName}
 
 	err := wait.PollUntilContextTimeout(context.Background(), 1*time.Second, 3*time.Minute, false, func(context context.Context) (bool, error) {
-		if err := kclient.Get(context, nsName, smcp); err != nil {
-			t.Logf("failed to get ServiceMeshControlPlane %s/%s: %v, retrying...", nsName.Namespace, nsName.Name, err)
+		if err := kclient.Get(context, nsName, istio); err != nil {
+			t.Logf("Failed to get Istio %s/%s: %v.  Retrying...", nsName.Namespace, nsName.Name, err)
 			return false, nil
 		}
-		if smcp.Status.Readiness.Components != nil {
-			pending := len(smcp.Status.Readiness.Components["pending"]) > 0
-			unready := len(smcp.Status.Readiness.Components["unready"]) > 0
-			if pending || unready {
-				t.Logf("found ServiceMeshControlPlane %s/%s, but it isn't ready. Retrying...", smcp.Namespace, smcp.Name)
-				return false, nil
-			}
-			if len(smcp.Status.Readiness.Components["ready"]) > 0 {
-				t.Logf("found ServiceMeshControlPlane %s/%s with ready components: %v", smcp.Namespace, smcp.Name, smcp.Status.Readiness.Components["ready"])
-				return true, nil
-			}
+		if istio.Status.GetCondition(sailv1.IstioConditionReady).Status == metav1.ConditionTrue {
```

Comment:
Aw, it no longer provides component status?  That was a great troubleshooting tool.